Content.Server/Chat/Systems/ChatSystem.cs

```diff
@@ -469,6 +470,16 @@ private string SanitizeMessageCapital(EntityUid source, string message)
         return message;
     }
 
+    private string SanitizeMessagePeriod(EntityUid source, string message)
```

Comment:
What do you need `source` for?

Reply:
Yeah on ```SanitizeMessageCapital``` it's the same thing for some reason. I'll remove it.